WordPress/gutenberg
# Storybook: Update to the version 6.5

## What?
Upgrade Storybook to v6.5

## Why?
Storybook 6.5 is out: https://storybook.js.org/blog/storybook-6-5/

## How?
Upgraded the storybook dependencies by running `npx storybook@latest upgrade`

## Testing Instructions
Run `npm run storybook:build` and `npm run storybook:dev`.

As far as I can see everything works as before
